gnucash.modules

```diff
@@ -147,7 +147,7 @@
 
   <autotools id="gwenhywfar" autogen-sh="configure"
              autogenargs="--with-guis='gtk3' --enable-local-install --disable-binreloc --disable-ssl PKG_CONFIG='pkg-config --dont-define-prefix'">
-    <branch module="223/gwenhywfar-4.99.25rc9.tar.gz" version="4.99.25rc9"
+    <branch module="228/gwenhywfar-5.0.0.tar.gz" version="5.0.0"
```

Comment:
https://www.aquamaniac.de/rdm/attachments/download/231/gwenhywfar-5.1.0.tar.gz is recent.